src/index.html

```diff
@@ -0,0 +1,273 @@
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+        </div>
+        <button class="section__more-btn" type="button">
+          <span>More</span>
+          <img src="./assets/icons/chevron-down.svg" alt="Chevron down icon" />
+        </button>
+      </div>
+    </section>
+    <section class="section section--dark">
+      <div class="container">
+        <h2 class="section__title">Partners</h2>
+        <div class="section__body">
+          <img
+            class="partner"
+            src="./assets/images/partner-1.png"
+            alt="Partner 1"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-2.png"
+            alt="Partner 2"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-3.png"
+            alt="Partner 3"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-4.png"
+            alt="Partner 4"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-5.png"
+            alt="Partner 5"
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      Event by
+      <img
+        height="40"
+        width="40"
+        src="./assets/images/git-nation.svg"
+        alt="Git Nation"
+      />
+      <strong>GitNation</strong>
+    </footer>
```

Comment:
- [ ] Kindly make sure that this section is not displayed for the mobile version as per the design and as suggested in the loom video attached.